### PR TITLE
Remove tag_exists check before starting a Brew image build

### PR DESCRIPTION
**Pull request by vfreex**
For the background, see discussions at
https://coreos.slack.com/archives/GDBRP5YJH/p1586479678450600?thread_ts=1586474171.448900&cid=GDBRP5YJH.

### CODE REVIEW

File: doozerlib/distgit.py [hunk 1]
```diff
@@ -736,59 +736,55 @@ def build_container(
...
+                        raise KeyboardInterrupt()
+
+                if len(profile["targets"]) > 1:
+                    # FIXME: Currently we don't really suppport building images against multiple targets,
+                    # or we would overwrite the image tag when pushing to the registry.
+                    # `targets` is defined as an array just because we want to keep consistency with RPM build.
+                    raise DoozerFatalError("Building images against multiple targets is not currently supported.")
+                target = '' if not profile["targets"] else profile["targets"][0]
+                exectools.retry(
+                    retries=retries, wait_f=wait,
```
Comment (sosiouxme) [inline doozerlib/distgit.py:784]: i think we still want this, to the extent that anyone might use `doozer --local` - we don't want to retry local builds.

File: doozerlib/distgit.py [hunk 2]
```diff
@@ -736,59 +736,55 @@ def build_container(
...
-                            raise KeyboardInterrupt()
-
-                    if len(profile["targets"]) > 1:
-                        # FIXME: Currently we don't really suppport building images against multiple targets,
-                        # or we would overwrite the image tag when pushing to the registry.
-                        # `targets` is defined as an array just because we want to keep consistency with RPM build.
-                        raise DoozerFatalError("Building images against multiple targets is not currently supported.")
-                    target = '' if not profile["targets"] else profile["targets"][0]
-                    exectools.retry(
-                        retries=(1 if self.runtime.local else retries), wait_f=wait,
```
Comment (sosiouxme) [inline doozerlib/distgit.py:788]: @vfreex Sorry, should have attached my previous comment to the "this" that I wanted to keep.
To the extent that anyone might use `doozer --local` - we don't want to retry local builds.

Comment (vfreex) [reply doozerlib/distgit.py:788]: @sosiouxme This is dead code and local build doesn't reach here. It is in the `else` branch of `if self.runtime.local:` https://github.com/openshift/doozer/pull/199/files#diff-8fa829dca0dd48a4369166e4f848476eL757

**Comment by vfreex** [comment]
Rebased.

File: doozerlib/distgit.py (hunk 2 above)
Comment (sosiouxme) [reply doozerlib/distgit.py:807]: I see. There probably shouldn't be an `else:` at all since that branch returns; but OK.

**Pull request merged**